fa/manual_pythonanywhere_deploy/README.md

```diff
@@ -0,0 +1,228 @@
+# Deploying to PythonAnywhere manually
```

Comment:
Is it intended that this file remained untranslated?

Reply:
Ive tried to translate it but I saw some issues and see that in es translation it is not included so I left it untranslated.